### PR TITLE
Dev: mavlink interface reg stream rates gets examples

### PR DESCRIPTION
**Pull request by rmackay9**
This adds cut-and-paste-able examples to the MAVLink Interface sections's [requesting data page](https://ardupilot.org/dev/docs/mavlink-requesting-data.html)

I've built the changes locally and tested the examples and it all seems fine

### CODE REVIEW

File: dev/source/docs/mavlink-requesting-data.rst [hunk 1]
```diff
@@ -135,6 +153,24 @@ Send a `COMMAND_LONG <https://mavlink.io/en/messages/common.html#COMMAND_LONG>`_
 - param2: time interval between messages in microseconds (i.e. 100000 for 10hz, 1000000 for 1hz)
 - param3 to param7: 0 (not used)
 
+**Examples**
+
+Here are some example commands that can be copy-pasted into MAVProxy (aka SITL) to test this command.  Before running these commands enter the following
+
+- module load message
```
Comment (peterbarker) [inline dev/source/docs/mavlink-requesting-data.rst:160]:
```suggestion
- module load messagerate
```

File: dev/source/docs/mavlink-requesting-data.rst [hunk 2]
```diff
@@ -135,6 +153,24 @@ Send a `COMMAND_LONG <https://mavlink.io/en/messages/common.html#COMMAND_LONG>`_
...
+
++---------------------------------------------------------+---------------------------------------------+
+| Example MAVProxy/SITL Command                           | Description                                 |
++=========================================================+=============================================+
+| ``message COMMAND_LONG 1 0 511 0 33 100000 0 0 0 0 0``  | Request GLOBAL_POSITION_INT at 10hz         |
```
Comment (peterbarker) [inline dev/source/docs/mavlink-requesting-data.rst:165]:
```suggestion
| ``messagerate set GLOBAL_POSITION_INT 10``  | Request GLOBAL_POSITION_INT at 10hz         |
```

similarly elsewhere

**Review by peterbarker — CHANGES_REQUESTED** [review]

File: dev/source/docs/mavlink-requesting-data.rst [hunk 3]
```diff
@@ -120,6 +120,24 @@ Send a `REQUEST_DATA_STREAM <https://mavlink.io/en/messages/common.html#REQUEST_
 - req_message_rate : the rate (in hz) of the message
 - start_stop : "1" to start sending, "0" to stop
 
+**Examples**
+
+Here are some example commands that can be copy-pasted into MAVProxy (aka SITL) to test this command.  Before running these commands enter the following
+
+- module load message
```
Comment (peterbarker) [inline dev/source/docs/mavlink-requesting-data.rst:127]:
```suggestion
- module load message
- set streamrate -1
```

File: dev/source/docs/mavlink-requesting-data.rst (hunk 1 above)
Comment (peterbarker) [inline dev/source/docs/mavlink-requesting-data.rst:160]:
```suggestion
- module load message
- set streamrate -1
```

**Review by peterbarker — DISMISSED** [review]

File: dev/source/docs/mavlink-requesting-data.rst (hunk 1 above)
Comment (peterbarker) [reply dev/source/docs/mavlink-requesting-data.rst:160]: Fair enough.

**Comment by Hwurzburg** [comment]
LGTM...thanks

**Pull request merged**